guide/ops/configuration/brooklyn_cfg.md

```diff
@@ -185,9 +185,23 @@ or
 [EntitlementManager]({{book.url.brooklyn_javadoc}}/org/apache/brooklyn/api/mgmt/entitlement/EntitlementManager.html).
 
 
-
 ## HTTPS Configuration
 
 See [HTTPS Configuration]({{book.path.docs}}/ops/configuration/https.md) for general information on configuring HTTPS.
 
 
+## Session configuration
+
+Apache Brooklyn uses a util class, `org.apache.brooklyn.rest.util.MultiSessionAttributeAdapter` for ensuring requests 
+in different bundles can get a consistent shared view of the data stored in the session.
+
+This class marks as used the session on the other modules by resetting the max inactive interval and also allows to 
+set up a max age time for the sessions, if not, it will be managed by Jetty the server.
```

Comment:
don't quite understand this - if not what?

Reply:
Hi Geoff, I added more detail. A larger explanation of the flow can be found on the javadoc of the class. I was trying to keep this documentation not to much connected to the internal java flow

Reply:
@jcabrerizo I think the issue is that it's missing a dot. The sentence should stops at `... age time for the sessions.` rather than continuing with a comma. 

Also at the end: `s/Jetty the server/the Jetty server`